docs/spec/Specifiers/Topology-Port-Specifiers.adoc

```diff
@@ -0,0 +1,73 @@
+=== Topology Port Specifiers
+
+A topology port specifiers names a <<Specifiers_Port-Instance-Specifiers,
+port instance>> on a <<Definitions_Topology-Definitions,
+topology definition>>. This allows topology connections to be made directly
+on a topology rather than referencing the instances named inside. It also
+allows topologies to <<Port-Interfaces_Binding,bind to port interfaces>>.
+
+==== Syntax
+
+`port`
+<<Lexical-Elements_Identifiers,_identifier_>>
+`=`
+<<Instance-Member-Identifiers_Port-Instance-Identifiers,_port-instance-id_>>
+
+==== Semantics
+
+For each topology port specifier defined in an enclosing topology _T_:
+
+. The _identifier_ names the port on the enclosing topology's
+<<Port-Interfaces_Anonymous-Port-Interfaces_Anonymous-Topology-Port-Interfaces,
+anonymous port interface>>.
+
+.. The _identifier_ must not conflict with any other topology port specifier in the
+enclosing topology.
+
+. The _port-instance-id_ references a <<Specifiers_Port-Instance-Specifiers,port instance>>
+endpoint in _T_.
+
+. The port referenced must be either a
+<<Specifiers_Port-Instance-Specifiers,output port instance>>
+or an
+<<Specifiers_Port-Instance-Specifiers,input port instance>>.
+
+.. Exporting an input or output port will expose a port instance of the same
+type to the enclosing topology _T_. Any connections made on this exposed port will
+internally be mapped to the port instance identifier _I_.
+
+. A topology port allows another topology that <<Specifiers_Topology-Import-Specifiers,
+imports>> this topology to connect directly to this named port.
+
+.. Providing a set of topology ports encapulates port instance names and component
+instance names from the interface of the parent topology.
+
+==== Example
+
+[source,fpp]
+----
+topology T {
+    instance c1
+    instance c2
+
+    connections {
+        # [...] Internal connections
```

Comment:
This notation is not clear. Does [...] mean that some code is being elided? If so I would move the ... out of the comment. What does "internal connections" mean? Are they connections between c1 and c2 and so "internal" to T? If so could you say that, or add some connections between c1 and c2?